sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -1677,11 +1677,22 @@ def get_test_params(cls, parameter_set="default"):
 
 
 class MeanAbsolutePercentageError(BaseForecastingErrorMetricFunc):
-    """Mean absolute percentage error (MAPE) or symmetric version.
+    r"""Mean absolute percentage error (MAPE) or symmetric version.
 
-    If `symmetric` is False then calculates MAPE and if `symmetric` is True
-    then calculates symmetric mean absolute percentage error (sMAPE). Both
-    MAPE and sMAPE output is non-negative floating point. The best value is 0.0.
+    For a univariate, non-hierarchical sample
+    of true values :math:`y_1, \dots, y_n` and
+    predicted values :math:`\widehat{y}_1, \dots, \widehat{y}_n`,
+    at time indices :math:`t_1, \dots, t_n`,
+    `evaluate` or call returns the Mean Absolute Percentage Error,
+    :math:`\frac{100\%}{n}\sum_{i=1}^n |\frac{y_i - \widehat{y}_i}{y_i}|`.
```

Comment:
I think it returns the metric in units of a float and not percentage, so we should replace the `100%` by `1` and the `200%` by `2`, right?

Might be worth adding a `Note: the metric returns the MAPE in fractional units, not percentages, that is, 0.5. rather than 50% etc`

Reply:
Yup. You are right! can't believe I miss that from the code.